### DISCUSSION TITLE
Documentation website for Bun.

### DISCUSSION THREAD

**Discussion by dulranga** [oven-sh/bun#331] [Ideas]
Bun doesn't have easy accessible documentation yet.
So can we build one with **docusaurus** ?

**Comment by marpit19** [comment]
For now we can refer to this: https://github.com/Jarred-Sumner/bun#Reference

**Reply by dulranga** [reply]
Yes, I would like to create a documentation website for this.
want to know about all you guys' ideas

**Reply by kosperera** [reply]
- GitHub Pages?
- Wiki on the repo?
- Docs folder in the repo? 

But, I believe, it would be quite easy to manage if bun has its own org. 

Related:
- Discussion #354
